### ISSUE TITLE
🛬 [INBOUND] - django-simple-history

### ISSUE THREAD

**Issue by ddabble** [django-commons/membership#131]
### What's the URL to the project?

https://github.com/jazzband/django-simple-history

#### Active:
@ddabble and @tim-schilling

#### Project leads:
@treyhunner and @rossmechanic

#### Other maintainers:
@jeking3 @thisisazeez @LeMinaw @valberg @brylie @johanneswilm @thesayfulla @mraza007

### Does the project have the required attributes?

- [x] Does the project have tests?
- [x] Does the project have documentation?
- [ ] Has the project adopted [Django Commons's Code of Conduct](https://github.com/django-commons/membership/blob/main/CODE_OF_CONDUCT.md)?
    - Will be done with https://github.com/jazzband/django-simple-history/pull/1429

### Who will be the new maintainers of the project?

@ddabble and @tim-schilling, as mentioned in https://github.com/jazzband/django-simple-history/issues/1391#issuecomment-2525288887

### Do you use test PyPI or want to test the packaging process?

- [x] Does the project have a test.pypi.org project?
    - Looks like it: https://test.pypi.org/project/django-simple-history/
- [x] Would you like to test your deployments with uploads to test.pypi.org (you'll need to create the project first)?
    - Sounds like something that could be useful, but I'm not familiar with this at all 😅

---

### Related Issues
* django-simple-history discussion issue: https://github.com/jazzband/django-simple-history/issues/1391 (mentioned above)
* Jazzband transfer issue: https://github.com/jazzband/help/issues/382

**Comment by ddabble** [comment]
Whoops, pressed enter too early 😅 Hold on while I edit the description

**Comment by tim-schilling** [comment]
@ddabble correct me if I'm wrong about the merge options:

```
"django-simple-history" = {
  description = "Store model history and view/revert changes from admin site. "
  allow_merge_commit = true # Allow merge commits on pull requests
  allow_rebase_merge = true # Allow rebase and merge commits on pull requests
  allow_squash_merge = true # Allow squash and merge commits on pull requests
  allow_update_branch = true # Allow updating source branch on pull requests
  admins = [
    # Include people who can release new versions
    "ddabble",
    "tim-schilling",
  ]
  committers = [
    # Include people who can commit to main / merge changes
  ]
  members = [
    # Include people who can assign/triage tickets
  ]
}
```

**Comment by tim-schilling** [comment]
@treyhunner any chance you have owner permissions for one or both of the the pypi.org and test.pypi.org projects?

**Comment by tim-schilling** [comment]
I should have clarified, can you add `cunla` (Daniel), `stormheg` (Storm) and `CodenameTim` (myself)? @ddabble if you have an account on those sites, can you please list it as well?

**Comment by ddabble** [comment]
> @ddabble correct me if I'm wrong about the merge options:
> [...]

(Am I understanding it correctly that this is some textual configuration of the repo settings?)
Sure, that looks good, though a personal preference is to not use rebase merging, as rebasing usually makes it more difficult getting a picture of which commits came from which PR. So I'd vote for setting `allow_rebase_merge = false`, but I'm fine with whatever 🙂

**Comment by ddabble** [comment]
> @ddabble if you have an account on those sites, can you please list it as well?

Yes, it's `dabble` on both sites :)

**Comment by tim-schilling** [comment]
You're understanding of that config is correct!

And interesting, I tend to like the rebase option because I find the merge commits a bit awkward and confusing compared to a linear history. I'm good with removing it though. It'll force me to try something new (my theme for 2025).

**Comment by treyhunner** [comment]
> I should have clarified, can you add `cunla` (Daniel), `stormheg` (Storm) and `CodenameTim` (myself)? We will take over ownership of the project on the PyPI sites. This would involve removing folks after 30 days who are not on the repo's administrators team (currently it's @ddabble and I).
> 
> @ddabble if you have an account on those sites, can you please list it as well?

I wasn't sure whether you needed maintainer or owner access. I've added `cunla`, `stormheg`, `dabble` as maintainers and `CodenameTim` as an owner. @tim-schilling feel free to increase the access privileges of all as needed.

Would `brianarmstrong`, `kyledseever`, `rossmechanic`, `ThePumpingLemma`, `macro1`, and `jazzband` (and possibly me) be removed from the PyPI after 30 days as well? Or would the project leads stay on PyPI also?

**Comment by Stormheg** [comment]
@treyhunner, I just received the PyPI invite - thank you 👍 

However, it permits us maintainer access only. Are you able to change our roles to owner?

**Comment by treyhunner** [comment]
> @treyhunner, I just received the PyPI invite - thank you 👍
> 
> However, it permits us maintainer access only. Are you able to change our roles to owner?

I was able to increase access for all but @ddabble. They'll need to accept the pending invite before someone can increase access from maintainer to owner.

**Comment by ddabble** [comment]
> > @treyhunner, I just received the PyPI invite - thank you 👍
> > 
> > However, it permits us maintainer access only. Are you able to change our roles to owner?
> 
> I was able to increase access for all but @ddabble. They'll need to accept the pending invite before someone can increase access from maintainer to owner.

Accepted now :)

**Comment by treyhunner** [comment]
> > > @treyhunner, I just received the PyPI invite - thank you 👍
> > > However, it permits us maintainer access only. Are you able to change our roles to owner?
> > 
> > 
> > I was able to increase access for all but @ddabble. They'll need to accept the pending invite before someone can increase access from maintainer to owner.
> 
> Accepted now :)

Just made you an owner 👍

**Comment by tim-schilling** [comment]
> Would brianarmstrong, kyledseever, rossmechanic, ThePumpingLemma, macro1, and jazzband (and possibly me) be removed from the PyPI after 30 days as well? Or would the project leads stay on PyPI also?

@treyhunner Yes, that would be the plan. Any of you are welcome to opt-in to staying on as an admin though! I don't believe anyone has over the few issues that exist, so @ddabble have been operating with the understanding that we would take over management of the repo.

**Comment by macro1** [comment]
it's been quite a while since i've been actively involved. happy to help if the need is there, but the project seems to be in capable hands with @ddabble and @rossmechanic before that. i'm fine to have my access decreased or removed if that's what you all think is best

**Comment by ddabble** [comment]
Thank you for your trust, @macro1, and also to you other guys for entrusting me with PyPI ownership! 😊

**Comment by tim-schilling** [comment]
@cunla do you want to take over from here? The PR to be squashed and merged (if it's good) is #221.

**Comment by ddabble** [comment]
Great work finishing the transfer, thank you @tim-schilling and @jezdez 😄

@tim-schilling Could you add me to the ReadTheDocs project? My username is `Dabble` 🙂

Also, should I be a member of our PyPI organization, or is that not necessary when I already have access to the PyPI project?

**Comment by tim-schilling** [comment]
@ddabble actually yes you should be a part of the PyPI project. What's your handle there?

Regarding ReadTheDocs, let me ask Trey. I don't have access to that.

**Comment by ddabble** [comment]
> Oh, you're already an owner 😅

Yes, I'm an owner of the project, but it doesn't seem like I'm a member of the PyPI organization..? When I visit https://pypi.org/manage/organizations/, it says "You have not joined any organizations on PyPI, yet.", at least 😅 Do you know what the benefits are of being in the PyPI organization? Might not be useful for anything 🤷

**Comment by ddabble** [comment]
Accepted the invitation to ReadTheDocs on Friday, so closing. Thanks for the help, Tim! 😊
